relay-event-normalization/src/validation.rs

```diff
@@ -44,7 +44,9 @@ pub fn validate_transaction(
 
     validate_transaction_timestamps(event, config)?;
     validate_trace_context(event)?;
-    validate_spans(event)?;
+    // There are no timestamp range requirements for span timestamps to reject
+    // the parent transaction.
```

Comment:
I would put this even more explicit. What do you think about this ? 
```suggestion
    // There are no timestamp range requirements for span timestamps.
    // Transaction will be rejected only if either end or start timestamp is missing. 
  
```